src/pages/index.tsx

```diff
@@ -1,24 +1,43 @@
 import CampaignHero from "../components/hero/campaignHero";
-import {
-  SkipNavigation,
-  TemplateAppContainer,
-} from "@nypl/design-system-react-components";
+import SwimLanes from "../components/swimlanes/swimLanes";
+import { TemplateAppContainer } from "@nypl/design-system-react-components";
+import data from "@/data/lanes";
+import { getNumItems } from "@/utils/utils";
 
-export default function Home() {
+export default function Home({ lanesWithNumItems }) {
```

Comment:
This seems to be really unwieldy code to pass this prop through this parent component. As far as I can tell, the solution would be to write a hook for `getNumItems` and move it inside the `swimLanes` component 
(` useEffect(() => {
    async function fetchData() {
      const lanes = data.lanes;` etc... )
But that doesn't work and I'm still unclear why.

Reply:
Yes, I understand the desire to not do prop drilling but because it's one level down, I don't think it's a big deal if the `lanesWithNumItems` gets passed from this `Home` component to the `SwimLanes` components. I'll continue by comment in the other thread.